flow/layers/performance_overlay_layer.h

```diff
@@ -30,6 +30,7 @@ class PerformanceOverlayLayer : public Layer {
                                    const char* font_path = nullptr);
 
   void Paint(PaintContext& context) const override;
+  void Preroll(PrerollContext* context, const SkMatrix& matrix) override;
```

Comment:
I wouldn't add this here

It's not very clear from any of the comments or documentation ;) -- but the PerformanceOverlayLayer is always supposed to draw on top of everything.  So we actually want the buggy behavior here

We also don't use this layer in practice on Fuchsia -- it's a debug tool -- so it probably isn't a big deal either way.